src/normalize-keyframes.js

```diff
@@ -243,6 +245,10 @@
           } else if (memberValue != 'replace') {
             throw new TypeError('Invalid composite mode ' + memberValue + '.');
           }
+        } else if (member == 'easing') {
+          console.log(memberValue);
+          debugger;
+          memberValue = shared.toTimingFunction(memberValue);
```

Comment:
Why has this conversion been moved from keyframe-interpolations.js to here? I don't think we want to convert from input value to function here, because this function is only keyframe normalization. By all means carry out the validation check, but I think we're at risk of converting to a function and storing it too early, which is what happened with the easing-TypeError issue we had before.